perfmetrics/scripts/install_bash.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script to install a specific version of GNU Bash to /usr/local/bin/bash
+# Usage: install_bash.go <bash-version>
+
+# Exit on error, treat unset variables as errors, and propagate pipeline errors.
+set -euo pipefail
```

Comment:
I believe you have added new scripts with execution permission.
`chmod +x script`